pkg/apiclient/requester.go

```diff
@@ -0,0 +1,62 @@
+package apiclient
+
+import (
+	"fmt"
+	"github.com/OctopusDeploy/cli/pkg/constants"
+	"github.com/spf13/cobra"
+	"runtime/debug"
+	"strings"
+)
+
+type Requester interface {
+	GetRequester() string
+}
+
+type RequesterContext struct {
+	cmd *cobra.Command
+}
+
+type FakeRequesterContext struct {
+}
+
+func NewRequester(c *cobra.Command) *RequesterContext {
+	return &RequesterContext{
+		cmd: c,
+	}
+}
+
+func (r *FakeRequesterContext) GetRequester() string { return "octopus/0.0.0" }
+
+func (r *RequesterContext) GetRequester() string {
+	version := "0.0.0"
```

Comment:
The easier way to get the CLI version is to use `version.Version` - it picks up the thing from version.txt so its basically a compile time constant (see version.go in the CLI source root folder)

Reply:
updated to read from the`version.txt` file via `version.Version`